app/views/layouts/application.html.haml

```diff
@@ -17,6 +17,9 @@
     %link{href: asset_path("govuk-apple-touch-icon-167x167.png"), rel: "apple-touch-icon", sizes: "167x167"}/
     %link{href: asset_path("govuk-apple-touch-icon-152x152.png"), rel: "apple-touch-icon", sizes: "152x152"}/
     %link{href: asset_path("govuk-apple-touch-icon.png"), rel: "apple-touch-icon"}/
+    %link{href: request.url, rel: "canonical"}/
```

Comment:
I’d prefer having the canonical link as only part of the vacancies listing page. From that call, I believe the intention is to add duplicate content for this and have them redirect to a single canonical version.

Reply:
That makes sense -  I will move this to the appropriate view

Reply:
Resolved this in d0d5072